sbws/sbws.py

```diff
@@ -7,6 +7,10 @@
 from sbws.util.config import validate_config
 from sbws.util.config import configure_logging
 from sbws.util.parser import create_parser
+from sbws import __version__ as SBWS_VERSION
+from stem import __version__ as STEM_VERSION
+from requests.__version__ import __version__ as REQ_VERSION
```

Comment:
i'd just not use capitals here, since __version__ is not in capitals.
I'd use ``as version`` for ``sbws`` version and ``stem_version``, ``requests_version``